CHANGELOG.md

```diff
@@ -1,8 +1,26 @@
-# Release Notes for Kubernetes 1.2.0
+<!--THIS TOC NEEDS TO BE GENERATED EVERY TIME THIS FILE IS UPDATED -->
```

Comment:
Hmm, ok, looks like you need to add munging of this doc:
https://github.com/kubernetes/kubernetes/blob/master/hack/after-build/update-generated-docs.sh#L42

Which would get us analytics etc, as well.